app/models/miq_server.rb

```diff
@@ -586,11 +586,20 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
-  def self.managed_resources
+  def self.managed_providers
+    ExtManagementSystem.group(:type).count
+  end
```

Comment:
Now that this method is much simpler, I think we can just inline it in the audit_details method.